mountainduck/preferences.md

```diff
@@ -43,15 +43,21 @@ Enable to prevent conflicting edits when accessing documents from a shared envir
 
 ### Mount Location 
 
-````{admonition} macOS only
+```{admonition} 
 :class: tip
 
+Only available on macOS using the standalone version.
+```
+
 Volumes are mounted in the *Volumes* folder in the [application support directory](support.md). You can change the default to another folder that is writable.
 
 **Note:** You **cannot** set it to `/Volumes` on macOS Mojave (10.14 or later) where the directory is not writable. The setting is not available in the version from the Mac App Store. 
 
 The default is set to `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Volumes.noindex`.
-````
+
+```{attention}
+Switching to the App Store version will cause permission issues if you customized the mount location.
```

Comment:
We should provide a note how to *resolve* such a problem instead. Thus make sure to reset the custom mount location.